Mathlib/CategoryTheory/Comma/Over.lean

```diff
@@ -620,7 +620,7 @@ variable {S : Type u₂} [Category.{v₂} S]
 /-- Given `X : T`, to upgrade a functor `F : S ⥤ T` to a functor `S ⥤ Over X`, it suffices to
     provide maps `F.obj Y ⟶ X` for all `Y` making the obvious triangles involving all `F.map g`
     commute. -/
-@[simps! obj_left map_left]
+@[simps! obj_left map_left obj]
```

Comment:
This change is unnecessary (the code compile without it) and problematic: one argument is that `obj_left` would no longer be triggered automatically. (Same below.)

Reply:
An earlier version of the constructions below needed it, but it was a bad fix anyway since it relied on the `obj` simp having higher priority than the `obj_left` simp